modules/nw-networkpolicy-about.adoc

```diff
@@ -12,7 +12,9 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
 
 [WARNING]
 ====
-Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules.
+Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules. However, if they are connected to pods with host network policies, they might be affected by the network policy rules.
+
+Pods currently cannot block traffic from localhost, nor from their resident nodes.
```

Comment:
Can you check that once from the K8s doc. Here they had mentioned - "The ability to prevent loopback or incoming host traffic (Pods cannot currently block localhost access, nor do they have the ability to block access from their resident node)."
https://kubernetes.io/docs/concepts/services-networking/network-policies/

Reply:
@subhtk It is OK if you want to use the sentence if the K8s doc is using it. But if you look at the bug itself the idea is to clarify the functionality of network policies. The traffic blocking is done either  network policies or the iptable rules on the node itself.

Reply:
Qualification could be - Loopback or incoming host traffic blocking might be done by implementing workarounds using Operating System components (such as SELinux, OpenVSwitch, IPTables, and so on) or Layer 7 technologies (Ingress controllers, Service Mesh implementations) or admission controllers.

Reply:
@asood-rh Acknowledged. I will change it to Network policies.